### DISCUSSION TITLE
keyboard input lag when unlocking vault (already reported upstream & fixed)

### DISCUSSION THREAD

**Discussion by distresscode** [dani-garcia/vaultwarden#2815] [Q&A] [answered]
As the title indicates, there's a lag of a few seconds when trying to unlock a vault. It seems this bug has been fixed per this discussion https://github.com/bitwarden/clients/issues/3562 and should be available in the latest release 2022.10. I'm using VW on Cloudron. 

Has this fix been implemented on VW? If so, I'll let the Cloudron team know there's a update. Thanks!

**Comment by stefan0xC** [comment]
Not sure if we had this issue. The `latest` release is still using web-vault v2022.6.2 and if it's fixed in v2022.10.0 than I've already made a PR: dani-garcia/bw_web_builds#84

Which client/server/web-vault version are you using?

**Comment by distresscode** [comment]
Hi @stefan0xC, here are the details:

App Title and Version: Vaultwarden 1.25.2
Package Version: com.github.bitwardenrs@1.9.2
Firefox v.105.0.3 (64-bit)
Bitwarden Firefox plugin v. 2022.9.1. There doesn't seem to be any updates for it.

**Comment by stefan0xC** [comment, marked as answer]
Ah, okay. If it's an issue with the browser extensions you'll have to wait for Bitwarden to release a new version. I don't think there is anything we can do about it as Vaultwarden is only the server (and the docker image ships a compatible web-vault version).

Which OS are you using because I don't have that issue (not with `vaultwarden/server:latest`/v1.25.2 nor with `vaultwarden/server:testing`/main branch) using Arch Linux and same Firefox version / Extension version.

**Reply by distresscode** [reply]
Happens on both of my machines, Win 10 & 11. In that case, I'll wait for a plugin update. Thank you!

**Comment by BlackDex** [comment]
This seems to be related to the browser extension, which has nothing to do with Vaultwarden or the web-vault build for Vaultwarden.

As @stefan0xC already mentioned, the stable release is still on the June release and i haven't seen this issue on the web-vault it self actually. I do see this on the extension.

**Reply by distresscode** [reply]
Noted. Thank you!
